src/improvements/tasks/sep/005-opcm-upgrade-v300/README.md

```diff
@@ -0,0 +1,40 @@
+# 005-opcm-upgrade-v300: Sepolia OPCM v3.0.0: op, soneium, ink
+
+Status: [DRAFT]()
+
+## Objective
+
+Executes [Upgrade 14](https://gov.optimism.io/t/upgrade-proposal-14-isthmus-l1-contracts-mt-cannon/9796) on Sepolia for the OP, Soneium, and Ink chains.
+
+In summary, this task uses `op-contract/v3.0.0` [OPContractsManager](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v3.0.0-rc.2/packages/contracts-bedrock/src/L1/OPContractsManager.sol) to upgrade 3 chains:
+1. OP Sepolia Testnet
+2. Soneium Testnet Minato
+3. Ink Sepolia
+
+### Timing
+
+Expected to be executed on or around 2025-04-09.
+
+## Transaction creation
+
+The transaction is generated by the [OPCMV300 template script](../../../template/OPCMUpgradeV300.sol),
+which reads the inputs from the [`config.toml`](./config.toml) file.
+
+## Signing and execution
+
+Follow the instructions in the [Nested Execution](../../../NESTED.md) guide for the following steps:
+
+- [1. Update repo](../../../NESTED.md#1-update-repo)
+- [2. Setup Ledger](../../../NESTED.md#2-setup-ledger)
+- [3. Simulate and validate the transaction](../../../NESTED.md#3-simulate-and-validate-the-transaction)
+
+Then follow the instructions in the [Validation](./VALIDATION.md) guide.
+
+## Simulation
+
+When simulating, ensure the logs say `Using script <your_path_to_superchain_ops>/superchain-ops/src/improvements/template/OPCMUpgradeV300.sol`.
+Navigate to the correct task directory then run the simulate command.
+```
+cd 004-opcm-upgrade-v300
+SIMULATE_WITHOUT_LEDGER=1 just --dotenv-path $(pwd)/.env --justfile ../../../nested.just simulate <foundation|council|chain-governor>
```

Comment:
You can remove chain-governor from this example command because this is the 2-of-2 (which doesn't include that role).